build_debian.sh

```diff
@@ -266,6 +266,10 @@ sudo mv $FILESYSTEM_ROOT/grub-pc-bin*.deb $FILESYSTEM_ROOT/$PLATFORM_DIR/x86_64-
 ## Disable kexec supported reboot which was installed by default
 sudo sed -i 's/LOAD_KEXEC=true/LOAD_KEXEC=false/' $FILESYSTEM_ROOT/etc/default/kexec
 
+## Modifty ntp default configuration: disable initial jump (add -x), and disable
+## jump when time difference is greater than 1000 seconds (remove -g).
+sudo sed -i 's/-g/-x/' $FILESYSTEM_ROOT/etc/default/ntp
```

Comment:
While this should be fine, I suggest making the regex more strict so as only to modify the `NTPD_OPTS` variable, just in case another line is ever added to this file.

Reply:
Great suggestion! Done.